{ 'stand_up_front_200': { 'head': [ { 'duration': 1.0,
                                      'name': u'stand_up_front_0800',
                                      'positions': [-0.0001, -0.795],
                                      'starttime': 4.005}],
                          'left_arm': [ { 'duration': 2.996,
                                          'name': u'stand_up_front_0600',
                                          'positions': [ -1.8601,
                                                         -1.3751,
                                                         -0.0074,
                                                         3.1316,
                                                         -1.8898,
                                                         1.0496,
                                                         -2.0087],
                                          'starttime': 0.0},
                                        { 'duration': 1.0,
                                          'name': u'stand_up_front_0700',
                                          'positions': [ -1.8029,
                                                         -1.6902,
                                                         0.0047,
                                                         3.1427,
                                                         -1.8841,
                                                         1.0217,
                                                         -1.9994],
                                          'starttime': 2.996},
                                        { 'duration': 2.998,
                                          'name': u'stand_up_front_0800',
                                          'positions': [ -2.2049,
                                                         -3.1235,
                                                         0.0018,
                                                         3.1418,
                                                         -1.871,
                                                         1.0295,
                                                         -1.995],
                                          'starttime': 3.996}],
                          'left_leg': [ { 'duration': 2.998,
                                          'name': u'stand_up_front_0600',
                                          'positions': [ 0.0005,
                                                         -0.0978,
                                                         -1.193,
                                                         1.2961,
                                                         1.8,
                                                         0.0],
                                          'starttime': 0.0},
                                        { 'duration': 1.0,
                                          'name': u'stand_up_front_0700',
                                          'positions': [ -0.0029,
                                                         -0.1004,
                                                         -1.5393,
                                                         1.605,
                                                         1.7954,
                                                         0.0036],
                                          'starttime': 2.998},
                                        { 'duration': 2.999,
                                          'name': u'stand_up_front_0800',
                                          'positions': [ 0.0022,
                                                         -0.1005,
                                                         -1.5483,
                                                         2.0048,
                                                         1.7941,
                                                         -0.0025],
                                          'starttime': 4.005}],
                          'right_arm': [ { 'duration': 3.0,
                                           'name': u'stand_up_front_0600',
                                           'positions': [ 1.8601,
                                                          1.3751,
                                                          0.0074,
                                                          -3.1316,
                                                          1.8898,
                                                          -1.0496,
                                                          2.0087],
                                           'starttime': 0.0},
                                         { 'duration': 1.0,
                                           'name': u'stand_up_front_0700',
                                           'positions': [ 1.8029,
                                                          1.6902,
                                                          -0.0047,
                                                          -3.1427,
                                                          1.8841,
                                                          -1.0217,
                                                          1.9994],
                                           'starttime': 3.0},
                                         { 'duration': 2.999,
                                           'name': u'stand_up_front_0800',
                                           'positions': [ 2.2049,
                                                          3.1235,
                                                          -0.0018,
                                                          -3.1418,
                                                          1.871,
                                                          -1.0295,
                                                          1.995],
                                           'starttime': 4.004}],
                          'right_leg': [ { 'duration': 2.999,
                                           'name': u'stand_up_front_0600',
                                           'positions': [ -0.0005,
                                                          0.0978,
                                                          1.193,
                                                          -1.2961,
                                                          -1.8,
                                                          -0.0],
                                           'starttime': 0.0},
                                         { 'duration': 1.0,
                                           'name': u'stand_up_front_0700',
                                           'positions': [ 0.0029,
                                                          0.1004,
                                                          1.5393,
                                                          -1.605,
                                                          -1.7954,
                                                          -0.0036],
                                           'starttime': 2.999},
                                         { 'duration': 3.0,
                                           'name': u'stand_up_front_0800',
                                           'positions': [ -0.0022,
                                                          0.1005,
                                                          1.5483,
                                                          -2.0048,
                                                          -1.7941,
                                                          0.0025],
                                           'starttime': 3.999}],
                          'torso': [ { 'duration': 2.998,
                                       'name': u'stand_up_front_0800',
                                       'positions': [-0.0, 1.2945],
                                       'starttime': 4.005}]}}
